<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>                             图像1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>                              图像2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gray_balance">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度均衡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grad_sharpen">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>梯度锐化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gray_hist">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>109</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度直方图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rgb_to_gray">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_images">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="laplacian_sharpen">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>拉普拉斯锐化</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>                            图像3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>                              图像4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="roberts_edge_detection">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Roberts边缘检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sobel_edge_detection">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel边缘检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="laplace_edge_detection">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Laplacian边缘检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prewitt_edge_detection">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prewitt边缘检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="canny_edge_detection">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Canny边缘检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_salt_noise">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>添加椒盐噪声</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_gaussian_noise">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>470</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>添加高斯噪声</string>
    </property>
   </widget>
   <widget class="QPushButton" name="average_filter">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>均值滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="middle_filter">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>中值滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="krisch_edge_detection">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Krisch边缘检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="window_filter">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>窗滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="form_filter">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>形态滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gaussian_filter">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>高斯滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="affine">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>210</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>仿射变换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="perspective">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>240</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>透视变换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threshold_seg">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>阈值分割</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OSTU">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>OSTU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Kittler">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Kittler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="frame_diff">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>帧间差分</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mix_guass">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>420</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>高斯混合背景</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circle_lbp">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>LBP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="target_det">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>直方图检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="model_check">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>模板匹配</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloaking">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>隐身效果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SIFT">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>240</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>SIFT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orb">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>270</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ORB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="color_fit">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>颜色匹配</string>
    </property>
   </widget>
   <widget class="QPushButton" name="svm_test">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>300</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>坐标点SVM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="word_test">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>字符测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Haar_1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Haar算法-水平</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Haar_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Haar算法-竖直</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gaber">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>180</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Garbor滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="face_haar">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>420</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>face_Haar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>111</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>图像预处理</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>111</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>边缘检测</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>噪声</string>
    </property>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>噪声</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>111</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>滤波</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>图像变换</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>280</y>
      <width>111</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>背景处理</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>111</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>特征明显</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>220</y>
      <width>111</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>特征提取</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camera2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>470</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像机标定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camera2_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>470</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>立体匹配</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>摄像标定</string>
    </property>
   </widget>
   <zorder>groupBox_10</zorder>
   <zorder>groupBox_8</zorder>
   <zorder>groupBox_7</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>cloaking</zorder>
   <zorder>label</zorder>
   <zorder>label_1</zorder>
   <zorder>gray_balance</zorder>
   <zorder>grad_sharpen</zorder>
   <zorder>gray_hist</zorder>
   <zorder>rgb_to_gray</zorder>
   <zorder>select_images</zorder>
   <zorder>laplacian_sharpen</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>roberts_edge_detection</zorder>
   <zorder>sobel_edge_detection</zorder>
   <zorder>laplace_edge_detection</zorder>
   <zorder>prewitt_edge_detection</zorder>
   <zorder>canny_edge_detection</zorder>
   <zorder>add_salt_noise</zorder>
   <zorder>add_gaussian_noise</zorder>
   <zorder>average_filter</zorder>
   <zorder>middle_filter</zorder>
   <zorder>krisch_edge_detection</zorder>
   <zorder>window_filter</zorder>
   <zorder>form_filter</zorder>
   <zorder>gaussian_filter</zorder>
   <zorder>affine</zorder>
   <zorder>perspective</zorder>
   <zorder>threshold_seg</zorder>
   <zorder>OSTU</zorder>
   <zorder>Kittler</zorder>
   <zorder>frame_diff</zorder>
   <zorder>mix_guass</zorder>
   <zorder>circle_lbp</zorder>
   <zorder>target_det</zorder>
   <zorder>model_check</zorder>
   <zorder>SIFT</zorder>
   <zorder>orb</zorder>
   <zorder>color_fit</zorder>
   <zorder>svm_test</zorder>
   <zorder>word_test</zorder>
   <zorder>Haar_1</zorder>
   <zorder>Haar_2</zorder>
   <zorder>gaber</zorder>
   <zorder>face_haar</zorder>
   <zorder>camera2</zorder>
   <zorder>camera2_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
